lib/shoryuken/manager.rb

```diff
@@ -13,22 +12,22 @@ def initialize(fetcher, polling_strategy)
       @queues = Shoryuken.queues.dup.uniq
 
       @done = Concurrent::AtomicBoolean.new(false)
-      @dispatching = Concurrent::AtomicBoolean.new(false)
 
       @fetcher = fetcher
       @polling_strategy = polling_strategy
 
-      @heartbeat = Concurrent::TimerTask.new(run_now: true,
```

Comment:
@waynerobinson you are right, maybe I was too overcautious. I added it in there just in case. But I'm considering removing it, `ensure` should work.